badge-maker/lib/color.js

```diff
@@ -4,15 +4,15 @@ const { fromString } = require('css-color-converter')
 
 // When updating these, be sure also to update the list in `badge-maker/README.md`.
 const namedColors = {
-  brightgreen: '#4c1',
-  green: '#97ca00',
-  yellow: '#dfb317',
-  yellowgreen: '#a4a61d',
-  orange: '#fe7d37',
-  red: '#e05d44',
+  brightgreen: '#10b73d',
+  green: '#67ac09',
```

Comment:
@JoshuaKGoldberg refs https://github.com/badges/shields/issues/5497#issuecomment-1902725815

Reply:
> Does `#67ac09` give enough contrast? I'm slightly unclear how you're accounting for the drop-shadow in the calculations

Heh, sorry, yeah - I wanted to explain it succinctly but don't think I succeeded.

Basically, I wanted to calculate the _average perceived color next to the text_ would be. It's hard in this case because of all the gradients at such small pixel sizes. Part guesswork, part math.

The math I used was basically going for the average of (darkest shadow color) and (color background).

IMO we've done enough gymnastics with colors for now. `#67ac09` is a definitive improvement over how it was before. I think it's good enough. If an expert comes in and indicates it's not, I don't think it'd be too difficult to switch to a darker color. Thoughts?